custom_components/ble_monitor/ble_parser/sensirion.py

```diff
@@ -0,0 +1,111 @@
+# Parser for Sensirion BLE advertisements
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def parse_sensirion(self, data, complete_local_name, source_mac, rssi):
+    """Sensirion parser"""
+    result = {"firmware": "Sensirion"}
+    sensirion_mac = source_mac
+    device_type = complete_local_name
+
+    if device_type != "MyCO2":
+        if self.report_unknown == "Sensirion":
+            _LOGGER.info(
+                "BLE ADV from UNKNOWN Sensirion DEVICE: RSSI: %s, MAC: %s, ADV: %s",
+                rssi,
+                to_mac(source_mac),
+                data.hex()
+            )
+        return None
+    
+    # check for MAC presence in sensor whitelist, if needed
+    if self.discovery is False and source_mac.lower() not in self.sensor_whitelist:
```

Comment:
Please remove the `.lower()` (was a bug in some other parsers, which will be fixed in the next release)